priority_alerts/serializers.py

```diff
@@ -0,0 +1,46 @@
+from rest_framework import serializers
+from rest_framework_gis import serializers as gis_serializers
+
+from priority_alerts import models
+
+
+class AlertsSerializers(gis_serializers.GeoFeatureModelSerializer):
+    """Serializer..."""
```

Comment:
docstring must say what it will do... not temporarily.

Reply:
Adjusted.